extensions/assets-manager/AssetsManagerEx.cpp

```diff
@@ -77,7 +77,7 @@ AssetsManagerEx::AssetsManagerEx(const std::string& manifestUrl, const std::stri
                                          int64_t totalBytesReceived,
                                          int64_t totalBytesExpected)
     {
-        this->onProgress(totalBytesExpected, totalBytesReceived, task.requestURL, task.identifier);
+        this->onProgress((double)totalBytesExpected, (double)totalBytesReceived, task.requestURL, task.identifier);
```

Comment:
I think it is more reasonable to modify `onProgress`. Why `onProgress` need to receive double parameters?

Reply:
I'm afraid that the download size is too big to store in a float.